projects/schematics/yarn.lock

```diff
@@ -1,1027 +1,2446 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
When I ran `yarn` on `projects/schematics` it generated a new `yarn.lock`. Can please double this?